amplify/src/main/java/com/github/stkent/amplify/tracking/AmplifyStateTracker.java

```diff
@@ -269,4 +184,16 @@ private boolean isEventAlreadyTracked(@NonNull final IEvent event) {
         return allTrackedEvents.contains(event);
     }
 
+    public AmplifyStateTracker defaultSettings() {
```

Comment:
Let's make this a static `getDefaultInstance` maybe, if we're thinking about how to allow multiple trackers in the future? Then you can either get the default instance or get a blank instance with regular ol' `getInstance` and do all the customization that way. Also note that the default instance is additive unless we add the ability to remove tracked events (not something to worry about yet, probably future)